provisioning/templates/etc/php/7.3/fpm/pool.d/www.conf

```diff
@@ -0,0 +1,441 @@
+; Start a new pool named 'www'.
```

Comment:
move file to 7.4 directory:  `provisioning/templates/etc/php/7.4/fpm/pool.d/www.conf`